code/___fulp_defines/fulp_defines.dm

```diff
@@ -54,13 +54,19 @@
 #define CLAN_VENTRUE "Ventrue Clan"
 /// Constant hallucinations & Bluespace Prophet traumas - Beefmen cannot join this.
 #define CLAN_MALKAVIAN "Malkavian Clan"
+/// Masters in shapeshifting and crafting creatures from flesh - forced to sleep in their claimed coffin ("Grounded").
+#define CLAN_TZIMISCE "Tzimisce Clan"
 /* Clan defines
 */
 #define TREMERE_SKELETON "Skeleton Pirate"
 #define TREMERE_ZOMBIE "Zombie - Revives"
 #define TREMERE_HUSK "Husk - Drinks all their Blood"
 #define TREMERE_BAT "Bat - Chance to give them Bat form"
 
+#define TZIMISCE_TWOARMED "Fast two-armed monster (150 Blood)"
+#define TZIMISCE_CLAWMONSTER "Clawed monster (250 Blood)"
+#define TZIMISCE_HUSK "Husk (+200 Blood)"
```

Comment:
why does this have a + in it?

also, uh, isn't the cost of a husk 150 points of blood?

the blood costs should really be their own defines